imgix.gemspec

```diff
@@ -1,7 +1,6 @@
-# coding: utf-8
```

Comment:
Can we confirm that this comment is unnecessary? Because there is something similar in imgix-python[[1](https://github.com/imgix/imgix-python/blob/7b3c92c27707aa0bc60288bb61448a0887a40bd7/imgix/urlhelper.py#L1)][[2](https://github.com/imgix/imgix-python#usage-with-utf-8)] and I'm wondering if it's encoding-related.

Reply:
Yeah, of course. The magic comment is used to alert ruby that the file contains non-ascii characters (which our's does not). Further, I am pretty sure this magic comment only specifies an encoding for the file it's declared in. I was first alerted to it being unnecessary from rubocop.

>The default script encoding is Encoding::UTF-8 after v2.0, but it can be changed by a magic comment on the first line of the source code file (or second line, if there is a shebang line on the first). [docs.ruby-lang.org](https://docs.ruby-lang.org/en/2.1.0/Encoding.html)

>The UTF-8 default encoding, which make many magic comments omissible... [Ruby 2.0 Release](https://www.ruby-lang.org/en/news/2013/02/24/ruby-2-0-0-p0-is-released/)

Here are a few Gemspec files that omit this magic comment: [rubocop](https://github.com/rubocop-hq/rubocop/blob/master/rubocop.gemspec), [rails](https://github.com/rails/rails/blob/master/rails.gemspec), [sequel](https://github.com/jeremyevans/sequel/blob/master/sequel.gemspec).

Reply:
Ah gotcha. All good then 👍